package.json

```diff
@@ -16,7 +16,13 @@
   "devDependencies": {},
   "scripts": {
     "start": "tsc server/ts/* --outDir server/js --lib es6,dom && tsc client/ts/* --outDir client/js --lib es6,dom && node server/js/main.js",
```

Comment:
I think it'd be nice to not have this in between state with two ways to build. Can we get rid of this now?

Reply:
yep, I'll remove the previous start command and replace it with the dev script